tests/DemosTest.php

```diff
@@ -234,7 +234,7 @@ protected function getPathWithAppVars(string $path): string
 
     public function demoFilesProvider(): array
     {
-        $excludeDirs = ['_demo-data', '_includes', '_unit-test', 'special'];
+        $excludeDirs = ['_demo-data', '_includes'];
```

Comment:
@ibelar FYI do not add anything else here, this is why https://github.com/atk4/ui/pull/1586 was not caught by tests before